js/app.jsx

```diff
@@ -412,11 +437,13 @@ var AddToSuite = React.createClass({
         <SearchableCaseSelectionList isNotIn={true} 
                                      suiteId={this.state.suite.id}
                                      onCheck={this.handleAdd}
+				     isCaseListUpdate={this.state.isCaseListUpdate}
```

Comment:
I wonder if we really need this prop? A prop in React should be a fixed thing, which should be immutable throughout the app's lifetime. A better way is to move it into the `state`, when triggered with `this.setState()`, the UI should be updated automatically.